app/javascript/components/users/user_show.jsx

```diff
@@ -51,25 +51,42 @@ class UserShowLandingPage extends React.Component {
           <Title content={"Musician Skill:"}/>
           <div style={{margin: 20}}>
             <Grid container direction="row" alignItems="flex-start">
-              
-              <Grid container xs={12} sm={6} justify="center" spacing={3}>
-                {user.approved_musician_skills.length === 0 &&
-                <SmallText content={"No approved skill"}/>
-                }
-                {user.approved_musician_skills.map((approved_musician_skill) =>
-                  <Grid item>
-                    <Badge color="rose"> <SmallText content={approved_musician_skill.name}/></Badge>
-                  </Grid>
-                )}
+              <Grid item xs={12} md={6}>
+                <Grid container justify="center" spacing={3}>
+                  {user.approved_musician_skills.length === 0 &&
+                  <SmallText content={"No approved skill"}/>
```

Comment:
Let's change all text for user to Russian? I know that it will be redone later with i18n, but anyway it is more logical to and Russian now